release: v1.49.0 (#5418)
apollographql/router: .changesets/fix_bnjjj_fix_5173.md

> ### Metrics overflow is logged as a warning rather than error ([Issue #5173](https://github.com/apollographql/router/issues/5173))

Reviewer: ```suggestion
### Log metrics overflow as a warning rather than an error ([Issue #5173](https://github.com/apollographql/router/issues/5173))
```

> If a metric has too high a cardinality the following warning will be displayed as a warning instead of an errorl:
> 

Reviewer: ```suggestion
If a metric has too high a cardinality, the following is displayed as a warning instead of an error:

```
